user/jwtValidator.go

```diff
@@ -0,0 +1,152 @@
+package user
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"strings"
+
+	auth0 "github.com/auth0-community/go-auth0"
+	jwt "github.com/dgrijalva/jwt-go"
+	jose "gopkg.in/square/go-jose.v2"
+)
+
+type JWTValidator interface {
+	ValidateRequest(request *http.Request) (*TokenData, error)
+}
+
+type jwtValidator struct {
+	config JWTValidatorConfig
+}
+
+type JWTValidatorConfig struct {
+	Auth0Domain string
+	Secret      string
+}
+
+func NewJWTValidator(config JWTValidatorConfig) (*jwtValidator, error) {
```

Comment:
it was more so I didn't have to change the signature as we won't need the secret but will likely add others such as audience ... but in saying that yeah it is a little redundant as it will move to platform